### DISCUSSION TITLE
Is there a way to enable the tool for arena open or qualifier weekend?

### DISCUSSION THREAD

**Discussion by JaggerMin** [unrealities/MTGA_Draft_17Lands#12] [Q&A] [answered]
I'm not sure if this is just me, but the tool doesn't recognize when I start a limited game in arena open or qualifier weekend. I'd love to make use of the tool in such large events!

**Comment by unrealities** [comment]
If anyone can save post-event logs, it might be trivial. However, I don't usually compete in them due to the expense.

**Comment by FiYir** [comment]
This app needs to see the set code and the word 'Draft' or 'Sealed' in the event name to detect the event.  I think the issue with the open, qualifier, and other events is that Arena doesn't include the set code in the event name. I don't compete in these events, but I've looked at the 17Lands event history from a user that has, and I'm seeing event names like 'OpenDraft_D2_Draft1_BO3' and 'OpenSealed_D1_Bo1'. The words 'Draft' and 'Sealed' appear in these names, but not the set code.

A solution would be to add an 'event_name' field to the temp_set_list entries and have the app use that when checking the events in the log. It would probably look something like the image below

![image](https://github.com/unrealities/MTGA_Draft_17Lands/assets/151777715/56fb2c86-cf4d-4337-8d27-1bba2e79742c)

I could work on this if someone could provide their Player.log from when they participated in the drafting portion of the event.

**Reply by JaggerMin** [reply]
[DraftLog_ONE_PremierDraft_a7399a38-2198-4052-9318-164ccd982bae.log](https://github.com/user-attachments/files/15525779/DraftLog_ONE_PremierDraft_a7399a38-2198-4052-9318-164ccd982bae.log)

Hey, would this do it? It's from the arena open day 2

**Comment by FiYir** [comment]
@JaggerMin could you go to the `Debug` folder, open the files in a text editor, search the files for any entry that reads `INFO - Event found`, and post the entry for the Arena Open? 

It will look similar to this 
![image](https://github.com/unrealities/MTGA_Draft_17Lands/assets/151777715/9a1b18ca-d95d-4b06-bce0-edb60311b1ea)

**Reply by JaggerMin** [reply]
The file for last month's arena open is long gone, but luckily today was MH3 arena open so I was able to get it's text


<06/23/2024 00:12:26> - INFO - Event found ArenaOpen_Day1_Bo1_20240622

also got the text for day 2:
<06/23/2024 22:39:43> - INFO - Event found ArenaOpen_Day2_DraftOne_20240623

**Reply by FiYir** [reply]
@JaggerMin Thanks.

**Comment by FiYir** [comment]
@JaggerMin [I added support for the Arena Open](https://github.com/unrealities/MTGA_Draft_17Lands/pull/23). The change will be in the next release.

Are you planning on participating in the qualifier tomorrow or the Arena Direct event next weekend?

**Comment by unrealities** [comment]
This has been released in v3.21. Please let us know if it is successful.

**Reply by JaggerMin** [reply]
![image](https://github.com/unrealities/MTGA_Draft_17Lands/assets/161361285/aacf0144-1cb3-4e0a-a123-d75db237e814)

I just started a qualifier event(the format is MH3 sealed Bo1), it doesn't seem to be register properly

**Reply by FiYir** [reply]
What's the event name in the debug file?

**Reply by JaggerMin** [reply]
<07/07/2024 23:13:28> - INFO - Event found QualifierPlayIn_Bo1_MH3_20240707

**Reply by FiYir** [reply, marked as answer]
Without restarting Arena, perform these steps:

1. Close the app
2. Open temp_set_list.json in the Temp folder
3. Delete the "Sealed" keyword from the "QualSealed" section near the bottom of the list, so it's just "Qualifier". It should look like the image below.
4. Open the app

![image](https://github.com/unrealities/MTGA_Draft_17Lands/assets/151777715/d609d7f9-6ef9-434c-a174-3839a73be9d3)

**Reply by JaggerMin** [reply]
![image](https://github.com/unrealities/MTGA_Draft_17Lands/assets/161361285/67b5270f-cec1-401d-b0f8-e3f9df6ee868)


Excellent! I can confirm it is successful

**Reply by FiYir** [reply]
Great! I will make a pull request for this change.

**Reply by unrealities** [reply]
This was updated in v3.22. Thanks for the quick work @FiYir !
